kde/plasma: 5.25.0 -> 5.25.1

andrevmatos opened the pull request:
###### Description of changes

Bump Plasma5 desktop to [5.25.1, released in 21 June 2022](https://kde.org/announcements/plasma/5/5.25.1/)

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

andrevmatos committed:
kde/plasma: 5.25.0 -> 5.25.1

andrevmatos committed:
Revert "powerdevil: fix brightness wonkiness on some laptops"

This reverts commit a29f64c22f331e1a1c650d4289dd891303c3766e.

andrevmatos commented:
Result of `nixpkgs-review pr 178500` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>181 packages marked as broken and skipped:</summary>
  <ul>
    <li>bluedevil</li>
    <li>breeze-grub</li>
    <li>breeze-gtk</li>
    <li>breeze-plymouth</li>
    <li>breeze-qt5</li>
    <li>buho</li>
    <li>clip</li>
    <li>communicator</li>
    <li>discover</li>
    <li>kactivitymanagerd</li>
    <li>kde-cli-tools</li>
    <li>kde-gtk-config</li>
    <li>kdenlive</li>
    <li>kdeplasma-addons</li>
    <li>kgamma5</li>
    <li>kinfocenter</li>
    <li>kmenuedit</li>
    <li>krohnkite</li>
    <li>kscreen</li>
    <li>kscreenlocker</li>
    <li>ksshaskpass</li>
    <li>kwallet-pam</li>
    <li>kwayland-integration</li>
    <li>kwin</li>
    <li>kwin-dynamic-workspaces</li>
    <li>kwin-tiling</li>
    <li>kwrited</li>
    <li>libsForQt512.applet-window-buttons</li>
    <li>libsForQt512.bismuth</li>
    <li>libsForQt512.bluedevil</li>
    <li>libsForQt512.breeze-grub</li>
    <li>libsForQt512.breeze-gtk</li>
    <li>libsForQt512.breeze-plymouth</li>
    <li>libsForQt512.breeze-qt5</li>
    <li>libsForQt512.buho</li>
    <li>libsForQt512.clip</li>
    <li>libsForQt512.communicator</li>
    <li>libsForQt512.discover</li>
    <li>libsForQt512.kactivitymanagerd</li>
    <li>libsForQt512.kde-cli-tools</li>
    <li>libsForQt512.kde-gtk-config</li>
    <li>libsForQt512.kde2-decoration</li>
    <li>libsForQt512.kdecoration</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.kgamma5</li>
    <li>libsForQt512.khotkeys</li>
    <li>libsForQt512.kinfocenter</li>
    <li>libsForQt512.kmenuedit</li>
    <li>libsForQt512.krohnkite</li>
    <li>libsForQt512.kscreen</li>
    <li>libsForQt512.kscreenlocker</li>
    <li>libsForQt512.ksshaskpass</li>
    <li>libsForQt512.ksystemstats</li>
    <li>libsForQt512.kwallet-pam</li>
    <li>libsForQt512.kwayland-integration</li>
    <li>libsForQt512.kwin</li>
    <li>libsForQt512.kwin-dynamic-workspaces</li>
    <li>libsForQt512.kwin-tiling</li>
    <li>libsForQt512.kwrited</li>
    <li>libsForQt512.layer-shell-qt</li>
    <li>libsForQt512.libkscreen</li>
    <li>libsForQt512.libksysguard</li>
    <li>libsForQt512.lightly</li>
    <li>libsForQt512.milou</li>
    <li>libsForQt512.nota</li>
    <li>libsForQt512.oxygen</li>
    <li>libsForQt512.parachute</li>
    <li>libsForQt512.pix</li>
    <li>libsForQt512.plasma-browser-integration</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt512.plasma-disks</li>
    <li>libsForQt512.plasma-integration</li>
    <li>libsForQt512.plasma-mobile</li>
    <li>libsForQt512.plasma-nano</li>
    <li>libsForQt512.plasma-nm</li>
    <li>libsForQt512.plasma-pa</li>
    <li>libsForQt512.plasma-sdk</li>
    <li>libsForQt512.plasma-systemmonitor</li>
    <li>libsForQt512.plasma-thunderbolt</li>
    <li>libsForQt512.plasma-vault</li>
    <li>libsForQt512.plasma-workspace</li>
    <li>libsForQt512.plasma-workspace-wallpapers</li>
    <li>libsForQt512.polkit-kde-agent</li>
    <li>libsForQt512.powerdevil</li>
    <li>libsForQt512.qqc2-breeze-style</li>
    <li>libsForQt512.sddm-kcm</li>
    <li>libsForQt512.shelf</li>
    <li>libsForQt512.spectacle</li>
    <li>libsForQt512.systemsettings</li>
    <li>libsForQt512.vvave</li>
    <li>libsForQt512.xdg-desktop-portal-kde</li>
    <li>libsForQt514.applet-window-buttons</li>
    <li>libsForQt514.bismuth</li>
    <li>libsForQt514.bluedevil</li>
    <li>libsForQt514.breeze-grub</li>
    <li>libsForQt514.breeze-gtk</li>
    <li>libsForQt514.breeze-plymouth</li>
    <li>libsForQt514.breeze-qt5</li>
    <li>libsForQt514.buho</li>
    <li>libsForQt514.clip</li>
    <li>libsForQt514.communicator</li>
    <li>libsForQt514.discover</li>
    <li>libsForQt514.kactivitymanagerd</li>
    <li>libsForQt514.kde-cli-tools</li>
    <li>libsForQt514.kde-gtk-config</li>
    <li>libsForQt514.kde2-decoration</li>
    <li>libsForQt514.kdecoration</li>
    <li>libsForQt514.kdenlive</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.kgamma5</li>
    <li>libsForQt514.khotkeys</li>
    <li>libsForQt514.kinfocenter</li>
    <li>libsForQt514.kmenuedit</li>
    <li>libsForQt514.krohnkite</li>
    <li>libsForQt514.kscreen</li>
    <li>libsForQt514.kscreenlocker</li>
    <li>libsForQt514.ksshaskpass</li>
    <li>libsForQt514.ksystemstats</li>
    <li>libsForQt514.kwallet-pam</li>
    <li>libsForQt514.kwayland-integration</li>
    <li>libsForQt514.kwin</li>
    <li>libsForQt514.kwin-dynamic-workspaces</li>
    <li>libsForQt514.kwin-tiling</li>
    <li>libsForQt514.kwrited</li>
    <li>libsForQt514.layer-shell-qt</li>
    <li>libsForQt514.libkscreen</li>
    <li>libsForQt514.libksysguard</li>
    <li>libsForQt514.lightly</li>
    <li>libsForQt514.milou</li>
    <li>libsForQt514.nota</li>
    <li>libsForQt514.oxygen</li>
    <li>libsForQt514.parachute</li>
    <li>libsForQt514.pix</li>
    <li>libsForQt514.plasma-browser-integration</li>
    <li>libsForQt514.plasma-desktop</li>
    <li>libsForQt514.plasma-disks</li>
    <li>libsForQt514.plasma-integration</li>
    <li>libsForQt514.plasma-mobile</li>
    <li>libsForQt514.plasma-nano</li>
    <li>libsForQt514.plasma-nm</li>
    <li>libsForQt514.plasma-pa</li>
    <li>libsForQt514.plasma-sdk</li>
    <li>libsForQt514.plasma-systemmonitor</li>
    <li>libsForQt514.plasma-thunderbolt</li>
    <li>libsForQt514.plasma-vault</li>
    <li>libsForQt514.plasma-workspace</li>
    <li>libsForQt514.plasma-workspace-wallpapers</li>
    <li>libsForQt514.polkit-kde-agent</li>
    <li>libsForQt514.powerdevil</li>
    <li>libsForQt514.qqc2-breeze-style</li>
    <li>libsForQt514.sddm-kcm</li>
    <li>libsForQt514.shelf</li>
    <li>libsForQt514.spectacle</li>
    <li>libsForQt514.systemsettings</li>
    <li>libsForQt514.vvave</li>
    <li>libsForQt514.xdg-desktop-portal-kde</li>
    <li>maui-nota</li>
    <li>milou</li>
    <li>oxygen</li>
    <li>plasma-browser-integration</li>
    <li>plasma-desktop</li>
    <li>plasma-integration</li>
    <li>plasma-mobile</li>
    <li>plasma-nano</li>
    <li>plasma-nm</li>
    <li>plasma-pa</li>
    <li>plasma-systemmonitor</li>
    <li>plasma-thunderbolt</li>
    <li>plasma-vault</li>
    <li>plasma-workspace</li>
    <li>plasma-workspace-wallpapers</li>
    <li>polkit-kde-agent</li>
    <li>powerdevil</li>
    <li>qqc2-breeze-style</li>
    <li>sddm-kcm</li>
    <li>shelf</li>
    <li>spectacle</li>
    <li>systemsettings</li>
    <li>vvave</li>
    <li>xdg-desktop-portal-kde</li>
  </ul>
</details>
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>kde-rounded-corners</li>
    <li>kdev-python</li>
    <li>libsForQt5.plasma-mobile</li>
  </ul>
</details>
<details>
  <summary>76 packages built:</summary>
  <ul>
    <li>digikam</li>
    <li>goverlay</li>
    <li>haruna</li>
    <li>kdev-php</li>
    <li>kdevelop</li>
    <li>kdevelop-unwrapped</li>
    <li>libsForQt5.applet-window-buttons</li>
    <li>libsForQt5.bismuth</li>
    <li>libsForQt5.bluedevil</li>
    <li>libsForQt5.breeze-grub</li>
    <li>libsForQt5.breeze-gtk</li>
    <li>libsForQt5.breeze-plymouth</li>
    <li>libsForQt5.breeze-qt5</li>
    <li>libsForQt5.buho</li>
    <li>libsForQt5.clip</li>
    <li>libsForQt5.communicator</li>
    <li>libsForQt5.discover</li>
    <li>libsForQt5.kactivitymanagerd</li>
    <li>libsForQt5.kde-cli-tools</li>
    <li>libsForQt5.kde-gtk-config</li>
    <li>libsForQt5.kde2-decoration</li>
    <li>libsForQt5.kdecoration</li>
    <li>libsForQt5.kdenlive</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.kgamma5</li>
    <li>libsForQt5.khotkeys</li>
    <li>libsForQt5.kinfocenter</li>
    <li>libsForQt5.kmenuedit</li>
    <li>libsForQt5.krohnkite</li>
    <li>libsForQt5.kscreen</li>
    <li>libsForQt5.kscreenlocker</li>
    <li>libsForQt5.ksshaskpass</li>
    <li>libsForQt5.ksystemstats</li>
    <li>libsForQt5.kwallet-pam</li>
    <li>libsForQt5.kwayland-integration</li>
    <li>libsForQt5.kwin</li>
    <li>libsForQt5.kwin-dynamic-workspaces</li>
    <li>libsForQt5.kwin-tiling</li>
    <li>libsForQt5.kwrited</li>
    <li>libsForQt5.layer-shell-qt</li>
    <li>libsForQt5.libkscreen</li>
    <li>libsForQt5.libksysguard</li>
    <li>libsForQt5.lightly</li>
    <li>libsForQt5.milou</li>
    <li>libsForQt5.nota</li>
    <li>libsForQt5.oxygen</li>
    <li>libsForQt5.parachute</li>
    <li>libsForQt5.pix</li>
    <li>libsForQt5.plasma-browser-integration</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5.plasma-disks</li>
    <li>libsForQt5.plasma-integration</li>
    <li>libsForQt5.plasma-nano</li>
    <li>libsForQt5.plasma-nm</li>
    <li>libsForQt5.plasma-pa</li>
    <li>libsForQt5.plasma-sdk</li>
    <li>libsForQt5.plasma-systemmonitor</li>
    <li>libsForQt5.plasma-thunderbolt</li>
    <li>libsForQt5.plasma-vault</li>
    <li>libsForQt5.plasma-workspace</li>
    <li>libsForQt5.plasma-workspace-wallpapers</li>
    <li>libsForQt5.polkit-kde-agent</li>
    <li>libsForQt5.powerdevil</li>
    <li>libsForQt5.qqc2-breeze-style</li>
    <li>libsForQt5.sddm-kcm</li>
    <li>libsForQt5.shelf</li>
    <li>libsForQt5.spectacle</li>
    <li>libsForQt5.systemsettings</li>
    <li>libsForQt5.vvave</li>
    <li>libsForQt5.xdg-desktop-portal-kde</li>
    <li>lightly-qt</li>
    <li>lxqt.lxqt-config</li>
    <li>material-kwin-decoration</li>
    <li>plasma-applet-volumewin7mixer</li>
    <li>sierra-breeze-enhanced</li>
    <li>wacomtablet</li>
  </ul>
</details>

hqurve commented:
Hi, thanks. Would you also mind including [this patch](https://invent.kde.org/frameworks/plasma-framework/-/commit/dff1b034c1162062aa2292099d3d01fc53dafdf6) for `plasma-framework`. It fixes thumbnails on icon-only task manager on X11. (https://bugs.kde.org/show_bug.cgi?id=455231)

andrevmatos committed:
kde-rounded-corners: bump to unstable, fix build with kwin 5.25

andrevmatos committed:
kdevelop: move to kde/gear release 22.04.2

Also, use python 3.9 to build kdev-python plugin, broken on python 3.10.

andrevmatos committed:
plasma-mobile: add plasma-workspace to buildInputs

andrevmatos committed:
kf5/plasma-framework: backport patch to fix thumbnails in task manager

andrevmatos commented:
Result of `nixpkgs-review pr 178500` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>345 packages marked as broken and skipped:</summary>
  <ul>
    <li>akregator</li>
    <li>bluedevil</li>
    <li>breeze-grub</li>
    <li>breeze-gtk</li>
    <li>breeze-plymouth</li>
    <li>breeze-qt5</li>
    <li>buho</li>
    <li>clip</li>
    <li>communicator</li>
    <li>discover</li>
    <li>dolphin</li>
    <li>dragon</li>
    <li>elisa</li>
    <li>gwenview</li>
    <li>k3b</li>
    <li>kactivitymanagerd</li>
    <li>kaddressbook</li>
    <li>kalendar</li>
    <li>kate</li>
    <li>kcalc</li>
    <li>kde-cli-tools</li>
    <li>kde-gtk-config</li>
    <li>kdenlive</li>
    <li>kdeplasma-addons</li>
    <li>kdev-php</li>
    <li>kdev-python</li>
    <li>kdevelop</li>
    <li>kdevelop-pg-qt</li>
    <li>kdevelop-unwrapped</li>
    <li>kdialog</li>
    <li>kexi</li>
    <li>kgamma5</li>
    <li>kget</li>
    <li>khelpcenter</li>
    <li>kinfocenter</li>
    <li>kmail</li>
    <li>kmenuedit</li>
    <li>kmix</li>
    <li>knights</li>
    <li>konsole</li>
    <li>kontact</li>
    <li>korganizer</li>
    <li>krohnkite</li>
    <li>krunner-symbols</li>
    <li>kscreen</li>
    <li>kscreenlocker</li>
    <li>ksshaskpass</li>
    <li>kwallet-pam</li>
    <li>kwayland-integration</li>
    <li>kwin</li>
    <li>kwin-dynamic-workspaces</li>
    <li>kwin-tiling</li>
    <li>kwrited</li>
    <li>libsForQt512.akonadi-calendar-tools</li>
    <li>libsForQt512.akonadi-import-wizard</li>
    <li>libsForQt512.akonadi-search</li>
    <li>libsForQt512.akonadiconsole</li>
    <li>libsForQt512.akregator</li>
    <li>libsForQt512.alkimia</li>
    <li>libsForQt512.applet-window-buttons</li>
    <li>libsForQt512.bismuth</li>
    <li>libsForQt512.bluedevil</li>
    <li>libsForQt512.breeze-grub</li>
    <li>libsForQt512.breeze-gtk</li>
    <li>libsForQt512.breeze-plymouth</li>
    <li>libsForQt512.breeze-qt5</li>
    <li>libsForQt512.buho</li>
    <li>libsForQt512.calendarsupport</li>
    <li>libsForQt512.clip</li>
    <li>libsForQt512.communicator</li>
    <li>libsForQt512.discover</li>
    <li>libsForQt512.dolphin</li>
    <li>libsForQt512.dolphin-plugins</li>
    <li>libsForQt512.dragon</li>
    <li>libsForQt512.elisa</li>
    <li>libsForQt512.eventviews</li>
    <li>libsForQt512.gwenview</li>
    <li>libsForQt512.incidenceeditor</li>
    <li>libsForQt512.k3b</li>
    <li>libsForQt512.kactivitymanagerd</li>
    <li>libsForQt512.kaddressbook</li>
    <li>libsForQt512.kalarm</li>
    <li>libsForQt512.kalendar</li>
    <li>libsForQt512.kate</li>
    <li>libsForQt512.kcalc</li>
    <li>libsForQt512.kclock</li>
    <li>libsForQt512.kde-cli-tools</li>
    <li>libsForQt512.kde-gtk-config</li>
    <li>libsForQt512.kde2-decoration</li>
    <li>libsForQt512.kdecoration</li>
    <li>libsForQt512.kdelibs4support</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt512.kdepim-addons</li>
    <li>libsForQt512.kdepim-runtime</li>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.kdev-php</li>
    <li>libsForQt512.kdev-python</li>
    <li>libsForQt512.kdevelop</li>
    <li>libsForQt512.kdevelop-pg-qt</li>
    <li>libsForQt512.kdevelop-unwrapped</li>
    <li>libsForQt512.kdialog</li>
    <li>libsForQt512.kgamma5</li>
    <li>libsForQt512.kget</li>
    <li>libsForQt512.khelpcenter</li>
    <li>libsForQt512.khotkeys</li>
    <li>libsForQt512.kinfocenter</li>
    <li>libsForQt512.kinit</li>
    <li>libsForQt512.kmail</li>
    <li>libsForQt512.kmail-account-wizard</li>
    <li>libsForQt512.kmenuedit</li>
    <li>libsForQt512.kmix</li>
    <li>libsForQt512.knights</li>
    <li>libsForQt512.knotes</li>
    <li>libsForQt512.konqueror</li>
    <li>libsForQt512.konsole</li>
    <li>libsForQt512.kontact</li>
    <li>libsForQt512.korganizer</li>
    <li>libsForQt512.kreport</li>
    <li>libsForQt512.krohnkite</li>
    <li>libsForQt512.krunner</li>
    <li>libsForQt512.krunner-symbols</li>
    <li>libsForQt512.kscreen</li>
    <li>libsForQt512.kscreenlocker</li>
    <li>libsForQt512.ksshaskpass</li>
    <li>libsForQt512.ksystemstats</li>
    <li>libsForQt512.kwallet-pam</li>
    <li>libsForQt512.kwayland-integration</li>
    <li>libsForQt512.kweather</li>
    <li>libsForQt512.kwin</li>
    <li>libsForQt512.kwin-dynamic-workspaces</li>
    <li>libsForQt512.kwin-tiling</li>
    <li>libsForQt512.kwrited</li>
    <li>libsForQt512.layer-shell-qt</li>
    <li>libsForQt512.libgravatar</li>
    <li>libsForQt512.libkdepim</li>
    <li>libsForQt512.libkscreen</li>
    <li>libsForQt512.libksieve</li>
    <li>libsForQt512.libksysguard</li>
    <li>libsForQt512.lightly</li>
    <li>libsForQt512.mailcommon</li>
    <li>libsForQt512.mailimporter</li>
    <li>libsForQt512.marble</li>
    <li>libsForQt512.mbox-importer</li>
    <li>libsForQt512.messagelib</li>
    <li>libsForQt512.milou</li>
    <li>libsForQt512.nota</li>
    <li>libsForQt512.oxygen</li>
    <li>libsForQt512.parachute</li>
    <li>libsForQt512.pim-data-exporter</li>
    <li>libsForQt512.pim-sieve-editor</li>
    <li>libsForQt512.pimcommon</li>
    <li>libsForQt512.pix</li>
    <li>libsForQt512.plasma-applet-caffeine-plus</li>
    <li>libsForQt512.plasma-applet-virtual-desktop-bar</li>
    <li>libsForQt512.plasma-browser-integration</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt512.plasma-disks</li>
    <li>libsForQt512.plasma-framework</li>
    <li>libsForQt512.plasma-integration</li>
    <li>libsForQt512.plasma-mobile</li>
    <li>libsForQt512.plasma-nano</li>
    <li>libsForQt512.plasma-nm</li>
    <li>libsForQt512.plasma-pa</li>
    <li>libsForQt512.plasma-sdk</li>
    <li>libsForQt512.plasma-settings</li>
    <li>libsForQt512.plasma-systemmonitor</li>
    <li>libsForQt512.plasma-thunderbolt</li>
    <li>libsForQt512.plasma-vault</li>
    <li>libsForQt512.plasma-workspace</li>
    <li>libsForQt512.plasma-workspace-wallpapers</li>
    <li>libsForQt512.polkit-kde-agent</li>
    <li>libsForQt512.powerdevil</li>
    <li>libsForQt512.print-manager</li>
    <li>libsForQt512.qqc2-breeze-style</li>
    <li>libsForQt512.qtcurve</li>
    <li>libsForQt512.sddm-kcm</li>
    <li>libsForQt512.shelf</li>
    <li>libsForQt512.soundkonverter</li>
    <li>libsForQt512.spectacle</li>
    <li>libsForQt512.systemsettings</li>
    <li>libsForQt512.vvave</li>
    <li>libsForQt512.xdg-desktop-portal-kde</li>
    <li>libsForQt512.yakuake</li>
    <li>libsForQt512.zanshin</li>
    <li>libsForQt514.akonadi-calendar-tools</li>
    <li>libsForQt514.akonadi-import-wizard</li>
    <li>libsForQt514.akonadi-search</li>
    <li>libsForQt514.akonadiconsole</li>
    <li>libsForQt514.akregator</li>
    <li>libsForQt514.alkimia</li>
    <li>libsForQt514.applet-window-buttons</li>
    <li>libsForQt514.bismuth</li>
    <li>libsForQt514.bluedevil</li>
    <li>libsForQt514.breeze-grub</li>
    <li>libsForQt514.breeze-gtk</li>
    <li>libsForQt514.breeze-plymouth</li>
    <li>libsForQt514.breeze-qt5</li>
    <li>libsForQt514.buho</li>
    <li>libsForQt514.calendarsupport</li>
    <li>libsForQt514.clip</li>
    <li>libsForQt514.communicator</li>
    <li>libsForQt514.discover</li>
    <li>libsForQt514.dolphin</li>
    <li>libsForQt514.dolphin-plugins</li>
    <li>libsForQt514.dragon</li>
    <li>libsForQt514.elisa</li>
    <li>libsForQt514.eventviews</li>
    <li>libsForQt514.gwenview</li>
    <li>libsForQt514.incidenceeditor</li>
    <li>libsForQt514.k3b</li>
    <li>libsForQt514.kactivitymanagerd</li>
    <li>libsForQt514.kaddressbook</li>
    <li>libsForQt514.kalarm</li>
    <li>libsForQt514.kalendar</li>
    <li>libsForQt514.kate</li>
    <li>libsForQt514.kcalc</li>
    <li>libsForQt514.kclock</li>
    <li>libsForQt514.kde-cli-tools</li>
    <li>libsForQt514.kde-gtk-config</li>
    <li>libsForQt514.kde2-decoration</li>
    <li>libsForQt514.kdecoration</li>
    <li>libsForQt514.kdelibs4support</li>
    <li>libsForQt514.kdenlive</li>
    <li>libsForQt514.kdepim-addons</li>
    <li>libsForQt514.kdepim-runtime</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.kdev-php</li>
    <li>libsForQt514.kdev-python</li>
    <li>libsForQt514.kdevelop</li>
    <li>libsForQt514.kdevelop-pg-qt</li>
    <li>libsForQt514.kdevelop-unwrapped</li>
    <li>libsForQt514.kdialog</li>
    <li>libsForQt514.kgamma5</li>
    <li>libsForQt514.kget</li>
    <li>libsForQt514.khelpcenter</li>
    <li>libsForQt514.khotkeys</li>
    <li>libsForQt514.kinfocenter</li>
    <li>libsForQt514.kinit</li>
    <li>libsForQt514.kmail</li>
    <li>libsForQt514.kmail-account-wizard</li>
    <li>libsForQt514.kmenuedit</li>
    <li>libsForQt514.kmix</li>
    <li>libsForQt514.knights</li>
    <li>libsForQt514.knotes</li>
    <li>libsForQt514.konqueror</li>
    <li>libsForQt514.konsole</li>
    <li>libsForQt514.kontact</li>
    <li>libsForQt514.korganizer</li>
    <li>libsForQt514.kreport</li>
    <li>libsForQt514.krohnkite</li>
    <li>libsForQt514.krunner</li>
    <li>libsForQt514.krunner-symbols</li>
    <li>libsForQt514.kscreen</li>
    <li>libsForQt514.kscreenlocker</li>
    <li>libsForQt514.ksshaskpass</li>
    <li>libsForQt514.ksystemstats</li>
    <li>libsForQt514.kwallet-pam</li>
    <li>libsForQt514.kwayland-integration</li>
    <li>libsForQt514.kweather</li>
    <li>libsForQt514.kwin</li>
    <li>libsForQt514.kwin-dynamic-workspaces</li>
    <li>libsForQt514.kwin-tiling</li>
    <li>libsForQt514.kwrited</li>
    <li>libsForQt514.layer-shell-qt</li>
    <li>libsForQt514.libgravatar</li>
    <li>libsForQt514.libkdepim</li>
    <li>libsForQt514.libkscreen</li>
    <li>libsForQt514.libksieve</li>
    <li>libsForQt514.libksysguard</li>
    <li>libsForQt514.lightly</li>
    <li>libsForQt514.mailcommon</li>
    <li>libsForQt514.mailimporter</li>
    <li>libsForQt514.marble</li>
    <li>libsForQt514.mbox-importer</li>
    <li>libsForQt514.messagelib</li>
    <li>libsForQt514.milou</li>
    <li>libsForQt514.nota</li>
    <li>libsForQt514.oxygen</li>
    <li>libsForQt514.parachute</li>
    <li>libsForQt514.pim-data-exporter</li>
    <li>libsForQt514.pim-sieve-editor</li>
    <li>libsForQt514.pimcommon</li>
    <li>libsForQt514.pix</li>
    <li>libsForQt514.plasma-applet-caffeine-plus</li>
    <li>libsForQt514.plasma-applet-virtual-desktop-bar</li>
    <li>libsForQt514.plasma-browser-integration</li>
    <li>libsForQt514.plasma-desktop</li>
    <li>libsForQt514.plasma-disks</li>
    <li>libsForQt514.plasma-framework</li>
    <li>libsForQt514.plasma-integration</li>
    <li>libsForQt514.plasma-mobile</li>
    <li>libsForQt514.plasma-nano</li>
    <li>libsForQt514.plasma-nm</li>
    <li>libsForQt514.plasma-pa</li>
    <li>libsForQt514.plasma-sdk</li>
    <li>libsForQt514.plasma-settings</li>
    <li>libsForQt514.plasma-systemmonitor</li>
    <li>libsForQt514.plasma-thunderbolt</li>
    <li>libsForQt514.plasma-vault</li>
    <li>libsForQt514.plasma-workspace</li>
    <li>libsForQt514.plasma-workspace-wallpapers</li>
    <li>libsForQt514.polkit-kde-agent</li>
    <li>libsForQt514.powerdevil</li>
    <li>libsForQt514.print-manager</li>
    <li>libsForQt514.qqc2-breeze-style</li>
    <li>libsForQt514.qtcurve</li>
    <li>libsForQt514.sddm-kcm</li>
    <li>libsForQt514.shelf</li>
    <li>libsForQt514.soundkonverter</li>
    <li>libsForQt514.spectacle</li>
    <li>libsForQt514.systemsettings</li>
    <li>libsForQt514.vvave</li>
    <li>libsForQt514.xdg-desktop-portal-kde</li>
    <li>libsForQt514.yakuake</li>
    <li>libsForQt514.zanshin</li>
    <li>marble</li>
    <li>maui-nota</li>
    <li>milou</li>
    <li>oxygen</li>
    <li>plasma-applet-caffeine-plus</li>
    <li>plasma-applet-virtual-desktop-bar</li>
    <li>plasma-browser-integration</li>
    <li>plasma-desktop</li>
    <li>plasma-integration</li>
    <li>plasma-mobile</li>
    <li>plasma-nano</li>
    <li>plasma-nm</li>
    <li>plasma-pa</li>
    <li>plasma-systemmonitor</li>
    <li>plasma-thunderbolt</li>
    <li>plasma-vault</li>
    <li>plasma-workspace</li>
    <li>plasma-workspace-wallpapers</li>
    <li>polkit-kde-agent</li>
    <li>powerdevil</li>
    <li>qqc2-breeze-style</li>
    <li>qtcurve</li>
    <li>sddm-kcm</li>
    <li>shelf</li>
    <li>spectacle</li>
    <li>systemsettings</li>
    <li>vvave</li>
    <li>xdg-desktop-portal-kde</li>
    <li>yakuake</li>
    <li>zanshin</li>
  </ul>
</details>
<details>
  <summary>172 packages built:</summary>
  <ul>
    <li>ajour</li>
    <li>amarok</li>
    <li>calligra</li>
    <li>colord-kde</li>
    <li>digikam</li>
    <li>goverlay</li>
    <li>haruna</li>
    <li>kaffeine</li>
    <li>kde-rounded-corners</li>
    <li>kdiff3</li>
    <li>kgraphviewer</li>
    <li>kile</li>
    <li>kmplayer</li>
    <li>kmymoney</li>
    <li>krename</li>
    <li>kronometer</li>
    <li>krunner-pass</li>
    <li>krusader</li>
    <li>kstars</li>
    <li>kup</li>
    <li>latte-dock</li>
    <li>libsForQt5.akonadi-calendar-tools</li>
    <li>libsForQt5.akonadi-import-wizard</li>
    <li>libsForQt5.akonadi-search</li>
    <li>libsForQt5.akonadiconsole</li>
    <li>libsForQt5.akregator</li>
    <li>libsForQt5.alkimia</li>
    <li>libsForQt5.applet-window-buttons</li>
    <li>libsForQt5.bismuth</li>
    <li>libsForQt5.bluedevil</li>
    <li>libsForQt5.breeze-grub</li>
    <li>libsForQt5.breeze-gtk</li>
    <li>libsForQt5.breeze-plymouth</li>
    <li>libsForQt5.breeze-qt5</li>
    <li>libsForQt5.buho</li>
    <li>libsForQt5.calendarsupport</li>
    <li>libsForQt5.clip</li>
    <li>libsForQt5.communicator</li>
    <li>libsForQt5.discover</li>
    <li>libsForQt5.dolphin</li>
    <li>libsForQt5.dolphin-plugins</li>
    <li>libsForQt5.dragon</li>
    <li>libsForQt5.elisa</li>
    <li>libsForQt5.eventviews</li>
    <li>libsForQt5.gwenview</li>
    <li>libsForQt5.incidenceeditor</li>
    <li>libsForQt5.k3b</li>
    <li>libsForQt5.kactivitymanagerd</li>
    <li>libsForQt5.kaddressbook</li>
    <li>libsForQt5.kalarm</li>
    <li>libsForQt5.kalendar</li>
    <li>libsForQt5.kate</li>
    <li>libsForQt5.kcalc</li>
    <li>libsForQt5.kclock</li>
    <li>libsForQt5.kde-cli-tools</li>
    <li>libsForQt5.kde-gtk-config</li>
    <li>libsForQt5.kde2-decoration</li>
    <li>libsForQt5.kdecoration</li>
    <li>libsForQt5.kdelibs4support</li>
    <li>libsForQt5.kdenlive</li>
    <li>libsForQt5.kdepim-addons</li>
    <li>libsForQt5.kdepim-runtime</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.kdev-php</li>
    <li>libsForQt5.kdev-python</li>
    <li>libsForQt5.kdevelop</li>
    <li>libsForQt5.kdevelop-pg-qt</li>
    <li>libsForQt5.kdevelop-unwrapped</li>
    <li>libsForQt5.kdialog</li>
    <li>libsForQt5.kgamma5</li>
    <li>libsForQt5.kget</li>
    <li>libsForQt5.khelpcenter</li>
    <li>libsForQt5.khotkeys</li>
    <li>libsForQt5.kinfocenter</li>
    <li>libsForQt5.kinit</li>
    <li>libsForQt5.kmail</li>
    <li>libsForQt5.kmail-account-wizard</li>
    <li>libsForQt5.kmenuedit</li>
    <li>libsForQt5.kmix</li>
    <li>libsForQt5.knights</li>
    <li>libsForQt5.knotes</li>
    <li>libsForQt5.konqueror</li>
    <li>libsForQt5.konsole</li>
    <li>libsForQt5.kontact</li>
    <li>libsForQt5.korganizer</li>
    <li>libsForQt5.kreport</li>
    <li>libsForQt5.krohnkite</li>
    <li>libsForQt5.krunner</li>
    <li>libsForQt5.krunner-symbols</li>
    <li>libsForQt5.kscreen</li>
    <li>libsForQt5.kscreenlocker</li>
    <li>libsForQt5.ksshaskpass</li>
    <li>libsForQt5.ksystemstats</li>
    <li>libsForQt5.kwallet-pam</li>
    <li>libsForQt5.kwayland-integration</li>
    <li>libsForQt5.kweather</li>
    <li>libsForQt5.kwin</li>
    <li>libsForQt5.kwin-dynamic-workspaces</li>
    <li>libsForQt5.kwin-tiling</li>
    <li>libsForQt5.kwrited</li>
    <li>libsForQt5.layer-shell-qt</li>
    <li>libsForQt5.libgravatar</li>
    <li>libsForQt5.libkdepim</li>
    <li>libsForQt5.libkscreen</li>
    <li>libsForQt5.libksieve</li>
    <li>libsForQt5.libksysguard</li>
    <li>libsForQt5.lightly</li>
    <li>libsForQt5.mailcommon</li>
    <li>libsForQt5.mailimporter</li>
    <li>libsForQt5.marble</li>
    <li>libsForQt5.mbox-importer</li>
    <li>libsForQt5.messagelib</li>
    <li>libsForQt5.milou</li>
    <li>libsForQt5.nota</li>
    <li>libsForQt5.oxygen</li>
    <li>libsForQt5.parachute</li>
    <li>libsForQt5.pim-data-exporter</li>
    <li>libsForQt5.pim-sieve-editor</li>
    <li>libsForQt5.pimcommon</li>
    <li>libsForQt5.pix</li>
    <li>libsForQt5.plasma-applet-caffeine-plus</li>
    <li>libsForQt5.plasma-applet-virtual-desktop-bar</li>
    <li>libsForQt5.plasma-browser-integration</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5.plasma-disks</li>
    <li>libsForQt5.plasma-framework</li>
    <li>libsForQt5.plasma-integration</li>
    <li>libsForQt5.plasma-mobile</li>
    <li>libsForQt5.plasma-nano</li>
    <li>libsForQt5.plasma-nm</li>
    <li>libsForQt5.plasma-pa</li>
    <li>libsForQt5.plasma-sdk</li>
    <li>libsForQt5.plasma-settings</li>
    <li>libsForQt5.plasma-systemmonitor</li>
    <li>libsForQt5.plasma-thunderbolt</li>
    <li>libsForQt5.plasma-vault</li>
    <li>libsForQt5.plasma-workspace</li>
    <li>libsForQt5.plasma-workspace-wallpapers</li>
    <li>libsForQt5.polkit-kde-agent</li>
    <li>libsForQt5.powerdevil</li>
    <li>libsForQt5.print-manager</li>
    <li>libsForQt5.qqc2-breeze-style</li>
    <li>libsForQt5.qtcurve</li>
    <li>libsForQt5.sddm-kcm</li>
    <li>libsForQt5.shelf</li>
    <li>libsForQt5.soundkonverter</li>
    <li>libsForQt5.spectacle</li>
    <li>libsForQt5.systemsettings</li>
    <li>libsForQt5.vvave</li>
    <li>libsForQt5.xdg-desktop-portal-kde</li>
    <li>libsForQt5.yakuake</li>
    <li>libsForQt5.zanshin</li>
    <li>lightly-qt</li>
    <li>lxqt.lxqt-config</li>
    <li>maia-icon-theme</li>
    <li>massif-visualizer</li>
    <li>material-kwin-decoration</li>
    <li>okteta</li>
    <li>partition-manager</li>
    <li>peruse</li>
    <li>plasma-applet-volumewin7mixer</li>
    <li>plasma-pass</li>
    <li>plasma-theme-switcher</li>
    <li>playbar2</li>
    <li>redshift-plasma-applet</li>
    <li>semantik</li>
    <li>sierra-breeze-enhanced</li>
    <li>skrooge</li>
    <li>syncthingtray</li>
    <li>systemdgenie</li>
    <li>wacomtablet</li>
    <li>zombietrackergps</li>
  </ul>
</details>

andrevmatos commented:
@SuperSandro2000 could you please give a look at this PR?

andrevmatos committed:
kio-extras: add optional dependencies

kactivities-stats enables "Recent Files"/"Recent
Locations"/"recentlyused:/" kioslaves, taglib enables audio metadata
reading, and libtirpc enables NFS kio.

SuperSandro2000 commented on pkgs/applications/kde/kdevelop/kdevelop.nix:
```diff
@@ -1,21 +1,18 @@
+, qtwebengine, cppcheck
 }:
 
+let
+  llvmPackages = llvmPackages_13;
+in
```
can we do this where the callPackage for this is? If not easily then this is also fine.

andrevmatos replied:
I tried it at first (to keep same interface as from the derivation before renaming), but its callPackage is in `kde/default.nix`, and since this is a newScope/function from `libsForQt5`, I couldn't find a `pkgs` reference there, so I had to hardcode the version. Also, couldn't find any other kde desktop/frameworks/gear package allowing overrides, to use as example. If you can point me to some reference, I'll be happy to implement it.

SuperSandro2000 commented on pkgs/applications/kde/kdevelop/kdev-python.nix:
```diff
@@ -1,13 +1,13 @@
+, threadweaver, ktexteditor, kdevelop-unwrapped, python39
+}:
+let
+  # FIXME: stick with py39 until
```
```suggestion
  # FIXME: use python 3.9 until 3.10 has less issues
```

andrevmatos replied:
This is not so much about py3.10 itself, but instead about kdev-python not yet having being updated; open MR url right below, just broke the line to avoid it being too long.

andrevmatos replied:
Adjusted comment to explain this better

andrevmatos commented on pkgs/development/libraries/kde-frameworks/plasma-framework.nix:
```diff
@@ -17,4 +17,12 @@ mkDerivation {
     qtquickcontrols2
   ];
   propagatedBuildInputs = [ kpackage kservice qtbase ];
+  patches = [
+    # FIXME: remove on kf5.96
+    (fetchpatch {
+      name = "fix-thumbnails-task-manager.patch";
+      url = "https://invent.kde.org/frameworks/plasma-framework/-/commit/dff1b034c1162062aa2292099d3d01fc53dafdf6.patch";
+      sha256 = "sha256-0162bi3J5bl5BmmUSrhxxy8MpLtSXkdHGK8wMcS5BB8=";
+    })
+  ];
```
Done

andrevmatos commented on pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix:
```diff
@@ -12,13 +12,13 @@
 stdenv.mkDerivation rec {
   pname = "kde-rounded-corners";
-  version = "0.1.1";
+  version = "20220617-g015ef5c";
```
Done and rebased PR on `master`
